containers/proxy-squid-image/squid.conf

```diff
@@ -2,6 +2,8 @@
 # To be used for Spacewalk Proxy servers.
 #
 
+/etc/squid/conf.d/*
```

Comment:
Nice! the good thing is that the user could mount their own config volume there. This is not made easy with the systemd services but it's possible.